### ISSUE TITLE
Undefined constant message

### ISSUE THREAD

**Issue by ksylvan** [leo-editor/leo-editor#3676]
#3334 is the same issue.  Fixed via PR #3716.

**EKR** Without `myLeoSettings.leo` I too get `expand_css_constants Unresolved @constants`.

<details><summary><b>Original Report</b></summary>
<br>

I came back here after a few years' absence! I love Leo and the philosophy!

I did an install on MacOS, with a `pip3 install leo`

```
kayvan@shiva ~ % which python3
/opt/homebrew/bin/python3
kayvan@shiva ~ % python3 -V
Python 3.11.6
```

And it seems to work.

When I run leo, it starts up (I see the hints window and the main window comes up)

On the console where I started Leo, I see this:

```
kayvan@shiva ~ % leo
setting leoID from os.getenv('USER'): 'kayvan'
Leo 6.7.5, stable branch, build 251a098fbf
2023-11-25 14:36:57 -0800
Python 3.11.6, PyQt version 5.15.11
darwin
load_session File part does not exist '/Users/kayvan/.leo/workbook.leo'
load_session Bad unl: 'unl:gnx:///Users/kayvan/.leo/workbook.leo#kayvan.20231127111148.2'
expand_css_constants Unresolved @constants
[
     0: '@screencast-font-style'
]

qt.qpa.fonts: Populating font family aliases took 147 ms. Replace uses of missing font family "DejaVu Sans Mono" with one that exists to avoid this cost.
unexpected error creating: None
Traceback (most recent call last):

  File "/opt/homebrew/lib/python3.11/site-packages/leo/commands/spellCommands.py", line 65, in create
    f = open(fn, mode='wb')
        ^^^^^^^^^^^^^^^^^^^

TypeError: expected str, bytes or os.PathLike object, not NoneType

expand_css_constants Unresolved @constants
[
     0: '@screencast-font-style'
]
```

What do I need to do to fix this?

<details>

**Comment by edreamleo** [comment]
@ksylvan My apologies for the delay in responding.

This issue is a bit of a mystery. Searches of leoSettings.leo show that an `@font screencast` node contains this:
```
screencast_font_family = Times New Roman
screencast_font_size = 18pt
screencast_font_weight = normal
screencast_font_style = normal
```
On my machine, Leo's `show-settings` command shows this:
```console
@color  screencastbg = yellow
@family screencastfontfamily = Times New Roman
@size   screencastfontsize = 18pt
@weight screencastfontweight = normal
```
As you can see, there is no entry for `screencastfontstyle`. But there is no warning either.

This looks like an obscure bug in Leo.

As a workaround, you might try adding `@string screencast-font-style = normal to your myLeoSettings.leo.

**Comment by edreamleo** [comment]
@ksylvan I get this message on Windows when I disable my myLeoSettings.leo. 

So this is not a MacOS issue. It's likely a plugin-specific issue.

**Comment by LewisNeal** [comment]
See #3334

**Comment by edreamleo** [comment]
Fixed via PR #3716.
